data/geany-3.0.css

```diff
@@ -1,10 +0,0 @@
-@import "geany.css";
-
-/* make close button on the editor's tabs smaller */
-#geany-close-tab-button {
-	-GtkWidget-focus-padding: 0;
-	-GtkWidget-focus-line-width: 0;
-	-GtkButton-default-border: 0;
-	-GtkButton-default-outside-border: 0;
-	-GtkButton-inner-border: 0;
-}
```

Comment:
> Lovely, they really like to break stuff in GTK+ don't they :)

I guess.  Funniest thing is that as a result (I guess?) it makes derived widgets not inherit the theme if they do set the name of their "style name".  So if you derive from `GtkButton` but set the name to "mybutton", you won't inherit the theme from "button".  And I'm not sure, but I don't think there is another way to allow specific theming apart adding custom style classes.
But who knows, they might change that in the future, breaking many apps in the process :)